modules/custom/az_person/config/install/views.view.az_person.yml

```diff
@@ -30,60 +30,12 @@ base_table: node_field_data
 base_field: nid
 display:
   default:
-    display_plugin: default
     id: default
     display_title: Master
+    display_plugin: default
     position: 0
     display_options:
-      access:
-        type: perm
-        options:
-          perm: 'access content'
-      cache:
-        type: none
-        options: {  }
-      query:
-        type: views_query
-        options:
-          disable_sql_rewrite: false
-          distinct: true
-          replica: false
-          query_comment: ''
-          query_tags: {  }
-      exposed_form:
-        type: basic
-        options:
-          submit_button: Apply
-          reset_button: false
-          reset_button_label: Reset
-          exposed_sorts_label: 'Sort by'
-          expose_sort_order: true
-          sort_asc_label: Asc
-          sort_desc_label: Desc
-      pager:
-        type: full
-        options:
-          items_per_page: 0
-          offset: 0
-          id: 0
-          total_pages: null
-      style:
-        type: views_bootstrap_grid
-        options:
-          grouping: {  }
-          row_class: ''
-          default_row_class: false
-          uses_fields: false
-          col_xs: col-12
```

Comment:
We should probably only be changing the view by adding the 3-col display, and not modifying the default display at all. Although I agree we need to fix the breakpoints at smaller screen sizes, that should probably be added as a separate PR so that it's more easily tracked as an intended change down the road. I would recommend reverting any changes to the default/master display and having those be follow-up issues so we can keep this PR as a view display addition only.